app/components/bookings/create/create-booking.component.html

```diff
@@ -0,0 +1,39 @@
+<ion-header>
+  <ion-navbar>
+    <ion-title>{{ _workpackage.stringId }} - {{ _workpackage.name }}</ion-title>
+  </ion-navbar>
+</ion-header>
+
+<ion-content>
+    <button accent floating block full *ngIf="!_bookingSession" (click)="createBookingSession()" >
+      <ion-icon name="ios-stopwatch-outline"></ion-icon>Livebuchung starten
+    </button>
+    <button accent floating block full *ngIf="_bookingSession" (click)="checkoutBookingSession()" >
+      <ion-icon name="ios-checkmark"></ion-icon>Checkout Session
+    </button>
+  <form [ngFormModel]="authForm" (ngSubmit)="onSubmit(authForm)">
+    <ion-item>
+      <ion-label>Datum</ion-label>
+      <ion-datetime displayFormat="DD.MM.YYYY" pickerFormat="DD MM YYYY" max="{{myDate}}"
+                    [(ngModel)]="myDate" [ngFormControl]="date" required></ion-datetime>
+    </ion-item>
+    <ion-item>
+      <ion-label>Aufwand</ion-label>
+      <ion-datetime displayFormat="HH:mm" pickerFormat="HH mm"
+                    [(ngModel)]="myEffort" [ngFormControl]="effort" (ionChange)="refreshEtc()"  required></ion-datetime>
+    </ion-item>
+    <ion-item class="item">
+      <ion-label floating >Beschreibung</ion-label>
+      <ion-textarea type="text" [ngFormControl]="description" required></ion-textarea>
+    </ion-item>
+    <ion-item>
+      <ion-label>ETC-aktuell:</ion-label>
+      <ion-label>{{ _workpackage.etc}}</ion-label>
```

Comment:
It seems counterintuitive, to pick an effort in an HH:MM format and then set the ETC as a float representing a fraction of a 8h workday. I think it would be a lot nicer, to display the current etc using the `WorkdaysToHoursPipe` and select the new ETC using the same kind of picker, as above (although we would probably have to raise the max value for hours).